modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -3065,6 +3065,101 @@ definition {
 		}
 	}
 
+	@description = "LPS-180195 - Verify on the Account object if it's possible to trigger an action with ADD when ADD an account"
```

Comment:
Use the story ticket for the description because we are testing the Story, not the testing task. LPS-173537